config/tsconfig.base.json

```diff
@@ -28,22 +28,47 @@
     "resolveJsonModule": true,
     "esModuleInterop": true,
     "sourceMap": true,
-    "types": ["node"],
+    "types": [
```

Comment:
Have you changed something in your formatter configuration? It's not the same as the output my code editor has.

Reply:
no, I haven't, but talking about that, the `source.organizeImports` vscode option conflicts with the lint sort rule
I think we should disable this lint rule once and for all...

Reply:
I support this idea 😄